e2e/main.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"time"
+)
+
+const urlsPath = "urls.json"
+const shortlyPath = "./shortly/shortly"
+const shortlyURL = "http://localhost:8080"
+
+func main() {
+	if err := run(); err != nil {
+		fmt.Fprintf(os.Stderr, "Error: %v\n", err)
+		os.Exit(1)
+	}
+}
+
+func run() error {
+	stg := &Storage{
+		URLPairs: []URLPair{URLPair{Short: "R7W7LQAA", Long: "golang.org"}},
+	}
+	if err := stg.SaveStorageFile(urlsPath); err != nil {
+		return fmt.Errorf("cannot init storage: %v", err)
+	}
+
+	// Start shortly server
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	cmd := exec.CommandContext(ctx, shortlyPath, "--load", urlsPath)
```

Comment:
As discussed it could be useful to redirect stdout+stderr of the child process into the parent